core/templates/dev/head/css/oppia.css

```diff
@@ -342,12 +342,16 @@ input.ng-dirty.ng-invalid, md-input-group.md-default-theme input.ng-dirty.ng-inv
 
 p {
   line-height: 1.2;
-  margin: 0;
+  margin-bottom: 18px;
   text-align: left;
   word-spacing: 0;
   word-break: break-word;
 }
 
+p:last-child {
```

Comment:
Hi @bansalnitish, @ankita240796 I think this will affect all paragraph tags on Oppia right? Is this intended? Just clarifying...

Reply:
Yes @nithusha21 , it affects the behaviour of all paragraph tags on Oppia but it provides a consistent behaviour as well. I checked for various pages and interactions and the change does not create any issues.